File: Arreglos/TiposEliminacion/IndiceEspecifico/IndiceEspecifico.cs
// Programa para eliminar un elemento en un índice específico en C#
// Autor: Rivera Cota Jesús Crispín
// FACULTAD DE INGENIERÍA LOS MOCHIS
// Ing. de Software 201

using System;
using System.Collections.Generic;

class Program {
    static void Main() {
        // Declaración e inicialización de la lista
        List<int> inputArr = new List<int> { 11, 21, 31, 41, 51, 61 };
        int position = 3; // Índice desde el que se realizará la eliminación

        // Imprimir el arreglo antes de la eliminación
        Console.WriteLine("Antes de la eliminación, el array es: ");
        for (int j = 0; j < inputArr.Count; j++) {
            Console.Write(inputArr[j] + " ");
        }

        // Eliminación del elemento en el índice indicado (posición 3)
        inputArr.RemoveAt(position);

        // Imprimir el arreglo después de la eliminación
        Console.WriteLine("\nDespués de la eliminación, el array es: ");
        for (int j = 0; j < inputArr.Count; j++) {
            Console.Write(inputArr[j] + " ");
        }
        Console.WriteLine();
    }
}

File: Arreglos/InserciónArreglos/IndiceEspecifico.java
// Inserción de un elemento en un índice específico en Java
// Autor: Rivera Cota Jesús Crispín
// FACULTAD DE INGENIERÍA LOS MOCHIS
// Ing. de Software 201

import java.util.ArrayList;
import java.util.Arrays;

public class IndiceEspecifico {
    public static void main(String[] args) {
        ArrayList<Integer> inputArr = new ArrayList<>(Arrays.asList(11, 21, 31, 41, 51, 61));
        int ele = 52;

        System.out.println("Antes de la inserción, el array es: ");
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }

        // Inserción del elemento en el tercer índice
        inputArr.add(3, ele);

        System.out.println("\nDespués de la inserción, el array es: ");
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }
        System.out.println();
    }
}
